includes/class-api.php

```diff
@@ -153,6 +153,20 @@ public function get_addons() {
 			];
 		}
 
+		// Catch responses that don't contain an array of add-ons.
+		if ( ! isset( $response['addons'] ) ) {
+			return [
+				'addons' => [],
```

Comment:
Should update the readme to show PHP 5.4+ syntax if that's what we are supporting.

Reply:
Added #21.